src/videojs-contrib-hls.js

```diff
@@ -48,7 +49,10 @@ const Hls = {
 };
 
 // the desired length of video to maintain in the buffer, in seconds
-Hls.GOAL_BUFFER_LENGTH = 30;
+Object.defineProperty(Hls, 'GOAL_BUFFER_LENGTH', {
+  get: () => SegmentLoader.GOAL_BUFFER_LENGTH,
+  set: (v) => SegmentLoader.GOAL_BUFFER_LENGTH = v
+});
```

Comment:
We heard you like getters so we put a getter inside your getter so you can get while you get